core/src/main/java/org/apache/accumulo/core/clientImpl/ScannerImpl.java

```diff
@@ -120,6 +120,15 @@ public ScannerImpl(ClientContext context, TableId tableId, Authorizations author
     this.size = Constants.SCAN_BATCH_SIZE;
   }
 
+  public ClientContext getClientContext() {
+    ensureOpen();
+    return context;
+  }
+
+  public TableId getTableId() {
+    return tableId;
```

Comment:
do we need  a call to `ensureOpen()` here for consistency?

Reply:
That would be good to do. Updated in ca2c510.